src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax.model;
+
+import static core.basesyntax.controller.ControllerHandler.checkArgsFailed;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.NoSuchElementException;
+
+public class User {
+    private static final String LOGIN_MATCHER = "[A-Za-z0-9\\d]+";
+    private static final int MIN_LOGIN_SIZE = 8;
+    private static final int MAX_LOGIN_SIZE = 20;
+    private final String login;
+    private final UserDto userData;
+
+    public User(String login, int age) {
+        this.login = login;
+        userData = new UserDto(age);
+    }
+
+    public static void checkPresence(User user) {
+        if (user == null) {
+            throw new NoSuchElementException("No such element found");
+        }
+    }
+
+    public static void loginCheck(String login) {
```

Comment:
avoid static methods, especially in model class

model class should have only getters/setters and so on